### ISSUE TITLE
[brainvision files] decoding/encoding error modifying brainvision .vhdr/.vmrk

### ISSUE THREAD

**Issue by dominikwelke** [mne-tools/mne-bids#154]
in my mne/mne_bids environment the util-functions `copyfile_brainvision` and `_get_brainvision_paths` break with errors since file encoding of .vhdr and .vmrk files is not specified. 

I'll open a pull request with modified code that checks and then specifies file encoding.

**Comment by sappelhoff** [comment]
hi @dominikwelke could you provide details on the file that you are using? Basically, how old is it / how was it produced, etc? Maybe provide the .vhdr and .vmrk files here?

Also, the error log would be appreciated!

Thanks for sending a PR to fix this already.

**Comment by dominikwelke** [comment]
sure, here is the full error message. 

```
---------------------------------------------------------------------------
UnicodeDecodeError                        Traceback (most recent call last)
<ipython-input-77-d70d65e4583c> in <module>
----> 1 mne_bids.write_raw_bids(raw, name_run, dir_run)

~/miniconda2/envs/mne/lib/python3.6/site-packages/mne_bids/mne_bids.py in write_raw_bids(raw, bids_basename, output_path, events_data, event_id, overwrite, verbose)
    710     # BrainVision is multifile, copy over all of them and fix pointers
    711     elif ext == '.vhdr':
--> 712         copyfile_brainvision(raw_fname, bids_fname)
    713     # EEGLAB .set might be accompanied by a .fdt - find out and copy it too
    714     elif ext == '.set':

~/miniconda2/envs/mne/lib/python3.6/site-packages/mne_bids/utils.py in copyfile_brainvision(vhdr_src, vhdr_dest)
    490                          ' but got "{}", "{}"'.format(ext_src, ext_dest))
    491 
--> 492     eeg_file_path, vmrk_file_path = _get_brainvision_paths(vhdr_src)
    493 
    494     # Copy data .eeg ... no links to repair

~/miniconda2/envs/mne/lib/python3.6/site-packages/mne_bids/utils.py in _get_brainvision_paths(vhdr_path)
    436     # Header file seems fine, read it
    437     with open(vhdr_path, 'r') as f:
--> 438         lines = f.readlines()
    439 
    440     # Try to find data file .eeg

~/miniconda2/envs/mne/lib/python3.6/encodings/ascii.py in decode(self, input, final)
     24 class IncrementalDecoder(codecs.IncrementalDecoder):
     25     def decode(self, input, final=False):
---> 26         return codecs.ascii_decode(input, self.errors)[0]
     27 
     28 class StreamWriter(Codec,codecs.StreamWriter):

UnicodeDecodeError: 'ascii' codec can't decode byte 0xc2 in position 693: ordinal not in range(128)
```

my python setup expects ascii decoding, while the file is actually utf-encoded

heres the .vhdr. unmodified, recorded 7.11.2018 with `BrainVision Recorder Professional - V. 1.21.0303`: [pilot_aeeeg_01.vhdr.zip](https://github.com/mne-tools/mne-bids/files/2680056/pilot_aeeeg_01.vhdr.zip)

**Comment by dominikwelke** [comment]
closed by #155
